kitsune/wiki/templates/wiki/includes/document_macros.html

```diff
@@ -46,7 +46,7 @@ <h1 class="title">{{ document.title }}</h1>
     {% if not fallback_reason %}
       {{ document.html|safe }}
     {% elif fallback_reason == 'no_translation' %}
-      {% if not document.is_archived %}
+      {% if document.is_localizable and document.any_localizable_revision() and not document.is_archived %}
```

Comment:
Here should be only `any_localizable_revision` rather than document.any_localizable_revision()